### PR TITLE
YEP-51-Update-profile-artist-route

### PR DESCRIPTION
**Pull request by alexis-moins**

### CODE REVIEW

File: back/src/composables/useUtils.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { Response } from 'express';
  * Enumeration containing all API messages return to the front-end.
  */
 enum ApiMessages {
-  //AUTH
+  // Auth
```
Comment (ASparton) [inline back/src/composables/useUtils.ts:7]: 👍

**Review by iittooss — APPROVED** [review]

File: back/src/v1/profile.ts [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  instagram_link: z.string().optional(),
+  facebook_link: z.string().optional(),
+  soundcloud_link: z.string().optional(),
+  youtube_link: z.string().optional(),
+  apple_music_link: z.string().optional(),
+  website_link: z.string().optional(),
+  deezer_link: z.string().optional(),
+
+  city_id: z.coerce.number().optional(),
+});
```
Comment (ASparton) [inline back/src/v1/profile.ts:27]: Add genres update?

File: back/src/v1/profile.ts [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+  website_link: z.string().optional(),
+  facebook_link: z.string().optional(),
+  instagram_link: z.string().optional(),
+
+  address: z.string().optional(),
+  city_id: z.coerce.number().optional(),
+
+  capacity_id: z.coerce.number().optional(),
+  host_type_id: z.coerce.number().optional(),
+});
```
Comment (ASparton) [inline back/src/v1/profile.ts:62]: Add genres update?

**Review by ASparton — CHANGES_REQUESTED** [review]
Add genres inside profile update routes

File: back/src/v1/profile.ts (hunk 2 above)
Comment (alexis-moins) [reply back/src/v1/profile.ts:27]: Sure!

File: back/src/v1/profile.ts (hunk 3 above)
Comment (alexis-moins) [reply back/src/v1/profile.ts:62]: Sure!

**Review by ASparton — APPROVED** [review]

**Pull request merged**